### DISCUSSION TITLE
CI for testing curl on ARM Architecture

### DISCUSSION THREAD

**Discussion by pheiduck** [curl/curl#8706] [Polls]
As Circle CI removed the arm hosts. We need a new CI to test cURL on ARM.
I think discussions is a better place to "discuss" about this topic.
https://github.com/curl/curl/pull/8630#issuecomment-1077675400

**Comment by bagder** [comment]
To me fosshost seems to be much more than "just" a CI platform, is it a practical solution to only use it for CI builds?

**Reply by pheiduck** [reply]
Some use this for arm Kernel building.
